reana_client/cli/workflow.py

```diff
@@ -271,15 +271,16 @@ def workflows_list(  # noqa: C901
 
         # Sort by given column, making sure that `None` is at the bottom of the list.
         def get_sort_key(x, column_id):
```

Comment:
Nice catch! It took me a while to understand what was going wrong.

Maybe to avoid this issues in the future we could avoid using `sort_column_name` inside of `get_sort_key` , and instead use `headers[column_id]`? In this way the "single source of truth" is the index, and the column name is only used to find the correct position in the array.

Also, to improve readability, we could maybe move the definition of `get_sort_key` to be just before the call to `sorted`?